kubetest/gke.go

```diff
@@ -330,11 +331,17 @@ func (g *gkeDeployer) Up() error {
 			}
 		}
 	}
-	// TODO(zmerlynn): The version should be plumbed through Extract
-	// or a separate flag rather than magic env variables.
-	if v := os.Getenv("CLUSTER_API_VERSION"); v != "" {
-		args = append(args, "--cluster-version="+v)
+
+	if *gkeReleaseChannel != "" {
+		args = append(args, "--release-channel="+*gkeReleaseChannel)
```

Comment:
If we wanted to make it run out of the box we need to enable autorepair if `rapid` channel is in use.

```go
		if strings.EqualFold(*gkeReleaseChannel, "rapid") {
			args = append(args, "--enable-autorepair")
		}
```

Technically we can put it in the prow job definition in `--gke-create-command` but it seems less user friendly.

Reply:
why? is this documented somewhere? What if rapid version changes, is this still true?

Reply:
I did not read about it anywher in documentation. But error message is quite self-descriptive:

```
ERROR: (gcloud.alpha.container.clusters.create) ResponseError: code=400, message=Auto_repair cannot be false when release_channel RAPID is set.                                                                   
```

Reply:
lol :see_no_evil:

Reply:
done